driver-mapping/src/main/java/com/datastax/driver/mapping/MappedUDTCodec.java

```diff
@@ -87,13 +87,35 @@ protected T deserializeAndSetField(ByteBuffer input, T target, String fieldName,
 
     @Override
     protected String formatField(T source, String fieldName) {
-        // This codec implementation is internal-use only, and its format method is never used
-        throw new UnsupportedOperationException();
+        if (!fieldName.startsWith("\""))
```

Comment:
I see where `formatField` is called in `AbstractUDTCodec#format` that the passed in fieldName is already escaped using `escapeId`, is surrounding in quotes needed?